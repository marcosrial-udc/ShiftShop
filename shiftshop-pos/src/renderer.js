import * as React from 'react';
import * as ReactDOM from 'react-dom';
import {Provider} from 'react-redux';
import {MuiThemeProvider} from '@material-ui/core';
import {SnackbarProvider} from 'notistack';
import {IntlProvider} from 'react-intl';

/* Async/await regenerator necesaries */
import 'core-js/stable';
import 'regenerator-runtime/runtime';

import './assets/scss/index.scss';
import theme from './theme'

import {App} from './modules/app';

import configureStore from './store';
import {initReactIntl} from "./i18n";
import './polyfills';

/* Configure store. */
const store = configureStore();

/* Configure i18n. */
const {locale, messages} = initReactIntl();

ReactDOM.render(
    <MuiThemeProvider theme={theme}>
      <Provider store={store}>
        <IntlProvider locale={locale} messages={messages}>
          <SnackbarProvider anchorOrigin={{vertical: 'top', horizontal: 'center'}}
                            autoHideDuration={2500}>
            <App/>
          </SnackbarProvider>
        </IntlProvider>
      </Provider>
    </MuiThemeProvider>,
    document.getElementById("root"));
