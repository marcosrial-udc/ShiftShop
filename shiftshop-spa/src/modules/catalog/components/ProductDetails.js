import React from 'react';
import {useSelector} from 'react-redux';
import PropTypes from 'prop-types';
import {FormattedDate, FormattedMessage} from 'react-intl';
import {Box, Chip, Grid} from '@material-ui/core';

import useStyles from '../styles/ProductDetails';

import * as selectors from '../selectors';
import {ProfitChip} from '../../common';
import ProductState from './ProductState';

const RowItem = ({label, content}) => {
    const classes = useStyles();

    return (
        <Grid className={classes.gridRow} container item spacing={1} alignItems="center">
            <Grid item xs={12} sm={4} lg={3}>
                <Box fontWeight={500} fontSize="body1.fontSize" fontStyle="italic">
                    <FormattedMessage id={label}/>
                </Box>
            </Grid>
            <Grid className={classes.rowContent} item xs>
                <Box fontSize="body1.fontSize">
                    {content}
                </Box>
            </Grid>
        </Grid>
    );
};

const ProductDetails = ({product}) => {

    const categories = useSelector(selectors.getCategories);

    return (
        <Grid container>
            <RowItem label="project.global.field.name" content={product.name}/>
            <RowItem label="project.global.field.category"
                     content={<Chip label={selectors.getCategoryName(categories, product.categoryId)}/>}/>
            {(product.providerPrice != null) &&
                <RowItem label="project.global.field.providerPrice" content={product.providerPrice.toFixed(2) + ' €'}/>}
            <RowItem label="project.global.field.salePrice" content={
                <Box display="flex" alignItems="center">
                    {product.salePrice.toFixed(2) + ' €'}
                    &nbsp;
                    {(product.providerPrice != null) && <ProfitChip profit={(product.salePrice - product.providerPrice)}/>}
                </Box>
            }/>
            <RowItem label="project.global.field.creationDate" content={<FormattedDate value={product.creationDate}/>}/>
            <RowItem label="project.global.field.barcode" content={<Box fontFamily="Menlo,Arial">{product.barcode}</Box>}/>
            <RowItem label="project.global.field.state"
                     content={product.active ? <ProductState active showMessage/> : <ProductState showMessage/>}/>
        </Grid>
    );

};

ProductDetails.propTypes = {
    product: PropTypes.object.isRequired
};

export default ProductDetails;
