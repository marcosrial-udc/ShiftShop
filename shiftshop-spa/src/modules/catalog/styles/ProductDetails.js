import {makeStyles} from '@material-ui/core';

const useStyles = makeStyles(theme => ({
    gridRow: {padding: theme.spacing(1)},
    rowContent: {
        [theme.breakpoints.down('sm')]:{
            marginLeft: theme.spacing(1),
        },
    }
}));

export default useStyles;
