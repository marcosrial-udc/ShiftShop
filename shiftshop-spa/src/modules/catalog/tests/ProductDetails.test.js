import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import {Provider} from 'react-redux';
import {IntlProvider} from 'react-intl';

import {initReactIntl} from "../../../i18n";

import ProductDetails from '../components/ProductDetails';

const {messages} = initReactIntl();

describe('ProductDetails snapshot', () => {

    const middlewares = [];
    const mockStore = configureStore(middlewares);

    const categories = {catalog: {categories: [{id: 1, name: 'Category'}]}};

    const createProductDetails = (store, product) => {

        return renderer.create(
            <Provider store={store}>
                <IntlProvider locale='en' messages={messages}>
                    <ProductDetails product={product}/>
                </IntlProvider>
            </Provider>
        );

    };

    test('Product active and with positive ROI', () => {

        // Initialize mockstore with empty state
        const initialState = categories;
        const store = mockStore(initialState);

        const product = {
            name: 'Product',
            categoryId: categories[0],
            providerPrice: 2.0,
            salePrice: 7.5,
            active: true
        };

        const productDetails = createProductDetails(store, product);

        expect(productDetails.toJSON()).toMatchSnapshot();

    });

    test('Product active and with neutral ROI', () => {

        // Initialize mockstore with empty state
        const initialState = categories;
        const store = mockStore(initialState);

        const product = {
            name: 'Product',
            categoryId: categories[0],
            providerPrice: 2.0,
            salePrice: 2.0,
            active: true
        };

        const productDetails = createProductDetails(store, product);

        expect(productDetails.toJSON()).toMatchSnapshot();

    });

    test('Product inactive and with negative ROI', () => {

        // Initialize mockstore with empty state
        const initialState = categories;
        const store = mockStore(initialState);

        const product = {
            name: 'Product',
            categoryId: categories[0],
            providerPrice: 2.0,
            salePrice: 1.75,
            active: false
        };

        const productDetails = createProductDetails(store, product);

        expect(productDetails.toJSON()).toMatchSnapshot();

    });

});
