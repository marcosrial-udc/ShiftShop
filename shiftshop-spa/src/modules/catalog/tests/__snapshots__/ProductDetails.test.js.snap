// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDetails snapshot Product active and with neutral ROI 1`] = `
<div
  className="MuiGrid-root MuiGrid-container"
>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-186"
      >
        Name
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-187"
      >
        Product
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-188"
      >
        Category
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-189"
      >
        <div
          className="MuiChip-root"
          onKeyDown={[Function]}
          onKeyUp={[Function]}
        >
          <span
            className="MuiChip-label"
          >
            
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-190"
      >
        Provider price
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-191"
      >
        2.00 €
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-192"
      >
        Sale price
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-193"
      >
        <div
          className="MuiBox-root MuiBox-root-194"
        >
          2.00 €
           
          <div
            className="MuiChip-root makeStyles-neutral-147 MuiChip-sizeSmall"
            onKeyDown={[Function]}
            onKeyUp={[Function]}
          >
            <span
              className="MuiChip-label MuiChip-labelSmall"
            >
              <div
                className="MuiBox-root MuiBox-root-195"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M10 17l5-5-5-5v10z"
                  />
                </svg>
                0.00
                 €
              </div>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-196"
      >
        Insertion date
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-197"
      >
        3/12/2020
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-198"
      >
        Barcode
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-199"
      >
        <div
          className="MuiBox-root MuiBox-root-200"
        />
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-201"
      >
        State
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-202"
      >
        <div
          className="MuiBox-root MuiBox-root-203 makeStyles-active-166"
        >
          <span
            className="MuiBadge-root makeStyles-statusBadge-168"
          >
            <span
              className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorPrimary"
            >
               
            </span>
          </span>
          <span
            className="makeStyles-statusMessage-169"
          >
            Active
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ProductDetails snapshot Product active and with positive ROI 1`] = `
<div
  className="MuiGrid-root MuiGrid-container"
>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-107"
      >
        Name
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-108"
      >
        Product
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-109"
      >
        Category
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-110"
      >
        <div
          className="MuiChip-root"
          onKeyDown={[Function]}
          onKeyUp={[Function]}
        >
          <span
            className="MuiChip-label"
          >
            
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-141"
      >
        Provider price
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-142"
      >
        2.00 €
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-143"
      >
        Sale price
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-144"
      >
        <div
          className="MuiBox-root MuiBox-root-145"
        >
          7.50 €
           
          <div
            className="MuiChip-root makeStyles-positive-146 MuiChip-sizeSmall"
            onKeyDown={[Function]}
            onKeyUp={[Function]}
          >
            <span
              className="MuiChip-label MuiChip-labelSmall"
            >
              <div
                className="MuiBox-root MuiBox-root-149"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 14l5-5 5 5z"
                  />
                </svg>
                5.50
                 €
              </div>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-159"
      >
        Insertion date
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-160"
      >
        3/12/2020
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-161"
      >
        Barcode
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-162"
      >
        <div
          className="MuiBox-root MuiBox-root-163"
        />
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-164"
      >
        State
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-165"
      >
        <div
          className="MuiBox-root MuiBox-root-170 makeStyles-active-166"
        >
          <span
            className="MuiBadge-root makeStyles-statusBadge-168"
          >
            <span
              className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorPrimary"
            >
               
            </span>
          </span>
          <span
            className="makeStyles-statusMessage-169"
          >
            Active
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ProductDetails snapshot Product inactive and with negative ROI 1`] = `
<div
  className="MuiGrid-root MuiGrid-container"
>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-204"
      >
        Name
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-205"
      >
        Product
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-206"
      >
        Category
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-207"
      >
        <div
          className="MuiChip-root"
          onKeyDown={[Function]}
          onKeyUp={[Function]}
        >
          <span
            className="MuiChip-label"
          >
            
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-208"
      >
        Provider price
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-209"
      >
        2.00 €
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-210"
      >
        Sale price
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-211"
      >
        <div
          className="MuiBox-root MuiBox-root-212"
        >
          1.75 €
           
          <div
            className="MuiChip-root makeStyles-negative-148 MuiChip-sizeSmall"
            onKeyDown={[Function]}
            onKeyUp={[Function]}
          >
            <span
              className="MuiChip-label MuiChip-labelSmall"
            >
              <div
                className="MuiBox-root MuiBox-root-213"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
                0.25
                 €
              </div>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-214"
      >
        Insertion date
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-215"
      >
        3/12/2020
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-216"
      >
        Barcode
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-217"
      >
        <div
          className="MuiBox-root MuiBox-root-218"
        />
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-gridRow-104 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-lg-3"
    >
      <div
        className="MuiBox-root MuiBox-root-219"
      >
        State
      </div>
    </div>
    <div
      className="MuiGrid-root makeStyles-rowContent-105 MuiGrid-item MuiGrid-grid-xs-true"
    >
      <div
        className="MuiBox-root MuiBox-root-220"
      >
        <div
          className="MuiBox-root MuiBox-root-221 makeStyles-inactive-167"
        >
          <span
            className="MuiBadge-root makeStyles-statusBadge-168"
          >
            <span
              className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary"
            >
               
            </span>
          </span>
          <span
            className="makeStyles-statusMessage-169"
          >
            Inactive
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
